example/networking/mellanox.com_v1alpha1_macvlannetwork_ipam-whereabouts_cr.yaml

```diff
@@ -14,7 +14,7 @@
 apiVersion: mellanox.com/v1alpha1
 kind: MacvlanNetwork
 metadata:
-  name: example-macvlannetwork
+  name: example-macvlannetwork-ipam-whereabouts
 spec:
   networkNamespace: "default"
   master: "ens2f0"
```

Comment:
can you set mtu to 9000 ?

Reply:
https://github.com/Mellanox/network-operator/blob/81f6a2260712389e99086d40ccdae7cbdef625ef/example/networking/rdma-net-cr-whereabouts-ipam.yml#L29